cmd/pyroscope/help.txt.tmpl

```diff
@@ -160,7 +160,7 @@ Usage of ./pyroscope:
   -querier.max-flamegraph-nodes-default int
     	Maximum number of flame graph nodes by default. 0 to disable. (default 8192)
   -querier.max-flamegraph-nodes-max int
-    	Maximum number of flame graph nodes allowed. 0 to disable.
+    	Maximum number of flame graph nodes allowed. 0 to disable. (default 1000000)
```

Comment:
Nit: Feels slightly inconsistent: Let's make it consistent either decimal or power of twos everywhere.  So `1<<20` for the max if we mimic the default with`1<<13`)